tempfork/gliderlabs/ssh/context.go

```diff
@@ -55,8 +55,6 @@ var (
 	// ContextKeyPublicKey is a context key for use with Contexts in this package.
 	// The associated value will be of type PublicKey.
 	ContextKeyPublicKey = &contextKey{"public-key"}
-
-	ContextKeySendAuthBanner = &contextKey{"send-auth-banner"}
```

Comment:
We don't need auth banner functionality in our gliderlabs fork anymore, but we still need the fork for some TTY-related changes.